boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common-pinctrl.dtsi

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <dt-bindings/pinctrl/nrf-pinctrl.h>
+
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 1, 6)>,
+				<NRF_PSEL(UART_RTS, 1, 5)>,
+				<NRF_PSEL(UART_RX, 1, 8)>,
+				<NRF_PSEL(UART_CTS, 1, 7)>;
```

Comment:
It should be: P1.05 for TX (37), P1.04 for RX (36), P1.07 for RTS (39), and P1.06 for RTS (38).